website/docs/concepts/scopes.mdx

```diff
@@ -0,0 +1,331 @@
+---
+title: Scopes
```

Comment:
A valuable feature to cover in this page would be `ProviderScope.parent`, combined with showing modals.

Reply:
Very good point. I actually ran into an issue with this API recently, when showing a custom overlay. [#1699](https://github.com/rrousselGit/riverpod/issues/1699)